tools/pipelines/build-bundle-size-artifacts.yml

```
@@ -26,8 +27,10 @@ extends:
     packageManagerInstallCommand: 'pnpm i --frozen-lockfile'
     packageManager: pnpm
     buildDirectory: .
-    tagName: bundle-artifacts
-    poolBuild: Large
+    tagName: bundle-artifact
```

Almost certainly needs to keep the s at the end
```suggestion
    tagName: bundle-artifacts
```